src/handlers/mentions.rs

```diff
@@ -94,23 +103,30 @@ pub(super) async fn handle_input(
         IssueData::load(&mut client, &event.issue, MENTIONS_KEY).await?;
     // Build the message to post to the issue.
     let mut result = String::new();
-    for to_mention in &input.paths {
-        if state.data.paths.iter().any(|p| p == to_mention) {
+    for to_mention in &input.to_mention {
+        if state.data.entries.iter().any(|p| p == to_mention) {
             // Avoid duplicate mentions.
             continue;
         }
-        let MentionsPathConfig { message, cc } = &config.paths[to_mention];
+        let MentionsEntryConfig { message, cc, type_ } = &config.entries[to_mention];
         if !result.is_empty() {
             result.push_str("\n\n");
         }
         match message {
             Some(m) => result.push_str(m),
-            None => write!(result, "Some changes occurred in {to_mention}").unwrap(),
+            None => match type_ {
+                MentionsEntryType::Filename => {
+                    write!(result, "Some changes occurred in {to_mention}").unwrap()
+                }
+                MentionsEntryType::Content => {
+                    write!(result, "Some changes regarding `{to_mention}` occurred").unwrap()
```

Comment:
We could maybe also print the filenames where the mention was found? In case it is a large diff. But I'm not sure if it's necessary, most of the time it should be obvious.